components/feature/downloads/src/test/java/mozilla/components/feature/downloads/SimpleDownloadDialogFragmentTest.kt

```diff
@@ -37,10 +37,12 @@ class SimpleDownloadDialogFragmentTest {
     @Before
     fun setup() {
         mockFragmentManager = mock()
+        val currentBytesCopied = ("Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko)" +
+                " Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36").length.toLong()
         download = DownloadState(
             "http://ipv4.download.thinkbroadband.com/5MB.zip",
             "5MB.zip", "application/zip", 5242880,
-            "Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36"
+            currentBytesCopied = currentBytesCopied
```

Comment:
nit: This value was for the `userAgent`, `currentBytesCopied` is an optional value, we don't have to provide it, if we don't need it

`userAgent = "Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36"`